policyengine_it/variables/household/person/is_dependent_between_18_and_21.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_it.model_api import *
+
+
+class is_dependent_between_18_and_21(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is dependent between 18 and 21 years old"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        return (
+            person("age", period) >= 21
```

Comment:
we would generally make those policy parameters unless the ages are defined by the Italian general legal code. We can make separate parameters for the '18' and '21' especially since there is also a limitation for children from 21 to 24

Since we define the child to be under 18 as well we might consider using the same parameter as in that file

For the naming if this is just used for a specific credit then we would name this 
`older_child_for_xyz_credit`  and store the parameters in the respective credit parameter folder